core/kazoo_voicemail/src/kvm_util.erl

```diff
@@ -370,6 +370,16 @@ maybe_transcribe(AccountId, MediaId, 'true') ->
     end;
 maybe_transcribe(_, _, 'false') -> 'undefined'.
 
+-spec transcribe_filename(kz_json:object()) -> kz_term:amy_ne_binary().
+transcribe_filename(MediaDoc) ->
+    TranscribeFilename = kzd_box_message:transcribe_filename(MediaDoc),
+    case TranscribeFilename =:= 'undefined'
+        andalso  kz_doc:attachment_names(MediaDoc) of
```

Comment:
Same thing here about not using the right side of `andalso` to return non-booleans